bin/migrate_vl_to_ldl.rb

```diff
@@ -0,0 +1,69 @@
+CSV_FILE_NAME = "vl_to_ldl_migration_#{Time.now.strftime('%Y%m%d')}.csv"
+HIV_PROGRAM = Program.where(name: "HIV PROGRAM").first
+
+# all test results with 1 as value numeric and = as value_modifier for HIV program
+def test_results
+  Observation.joins(:encounter).where(encounter: {program: HIV_PROGRAM }, obs: {value_modifier: '=', value_numeric: 1})
```

Comment:
Can we be specific with the type of test here. we only want to handle HIV Viral Load